Windows CI builds are broken - `$ENV{SHELL}` path expands Win-style

Most probably an update of the VM image. 

Issue:
```
[ 18%] Building CXX object external/iv/src/lib/CMakeFiles/interviews.dir/InterViews/slider.cpp.obj
The system cannot find the file specified
make[2]: *** [src/nrniv/CMakeFiles/nrnversion_h.dir/build.make:70: src/nrniv/CMakeFiles/nrnversion_h] Error 1
make[1]: *** [CMakeFiles/Makefile2:466: src/nrniv/CMakeFiles/nrnversion_h.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

```
2022-01-19T01:14:31.7843905Z The system cannot find the file specified
2022-01-19T01:14:31.7955241Z make[2]: *** [src/nrniv/CMakeFiles/nrnversion_h.dir/build.make:70: src/nrniv/CMakeFiles/nrnversion_h] Error 1
2022-01-19T01:14:31.7977241Z make[1]: *** [CMakeFiles/Makefile2:466: src/nrniv/CMakeFiles/nrnversion_h.dir/all] Error 2
```
Definitely a puzzle. The ```build/src/nrniv/CMakeFiles/nrnversion_h.dir``` folder exists on my last build in my Windows virtualbox guest but CMake seems to have a different build strategy on Linux and that folder does not exist in my linux builds.

--- a/src/nrniv/CMakeLists.txt
+++ b/src/nrniv/CMakeLists.txt
@@ -145,7 +145,7 @@ set(NRN_INCLUDE_DIRS
 # ~~~
 add_custom_target(
   nrnversion_h
-  COMMAND ${CMAKE_COMMAND} -E env PROJECT_VERSION=${PROJECT_VERSION} $ENV{SHELL} ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh ${PROJECT_SOURCE_DIR} > nrnversion.h.tmp
+  COMMAND ${CMAKE_COMMAND} -E env PROJECT_VERSION=${PROJECT_VERSION} bash ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh ${PROJECT_SOURCE_DIR} > nrnversion.h.tmp
   COMMAND ${CMAKE_COMMAND} -E copy_if_different nrnversion.h.tmp ${NRN_NRNOC_SRC_DIR}/nrnversion.h
   DEPENDS ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh)
 

